uicomponent-compose/main/src/main/java/io/github/droidkaigi/feeder/DrawerContent.kt

```diff
@@ -48,26 +59,26 @@ enum class DrawerContents(
     ),
     ABOUT_DROIDKAIGI(
         group = Group.OTHER,
-        imageResId = R.drawable.ic_baseline_list_alt_24,
+        imageResId = R.drawable.ic_baseline_android_24,
         label = "DroidKaigiとは",
```

Comment:
Is this wording correct?
In Figma, it says `このアプリとは`, but now it says `DroidKaigiとは`.
At DroidKaigi 2020 it will be `DroidKaigiとは`.

I like `DroidKaigiとは` better!

Reply:
For the time being, I think this is good for this PR. Thank you for thinking and implementing it instead of implementing it as it is!
@nontan Do you have any opinions?

Reply:
Sorry, I talked to the Designer in this part, I was told that English could be used in a unified way.
So I want to use "About DroidKaigi" and English for others.

@nontan Would you please change it to English in Figma?
このアプリとは -> About DroidKaigi
コントリビューター -> Contributor
スタッフ -> Staff
設定 -> Setting

Reply:
If you think about it, you don't have to deal with this PR, so you don't have to.

Reply:
I'm sorry for the late reply. I changed it!